cairo-contracts/packages/apps/src/transfer/types.cairo

```diff
@@ -102,6 +102,15 @@ pub impl PrefixedDenomImpl of PrefixedDenomTrait {
         }
     }
 
+    fn last_prefix(self: @PrefixedDenom) -> Option<@TracePrefix> {
```

Comment:
I see it as `first_prefix` -- as we maintain the trace_path backwards. but it's up to you.

Reply:
Hmm, this is a bit tricky. But let's go with yours. [482c98f](https://github.com/informalsystems/ibc-starknet/pull/326/commits/482c98f97c0fc01080f639b5e196ed2673dc9476)